[2025-03-01T02:09:35.200+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-01T02:09:35.222+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_postgres_1.generate_orders_data manual__2025-03-01T02:09:28.973488+00:00 [queued]>
[2025-03-01T02:09:35.232+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_postgres_1.generate_orders_data manual__2025-03-01T02:09:28.973488+00:00 [queued]>
[2025-03-01T02:09:35.233+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-01T02:09:35.248+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): generate_orders_data> on 2025-03-01 02:09:28.973488+00:00
[2025-03-01T02:09:35.258+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=58556) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-01T02:09:35.258+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_postgres_1', 'generate_orders_data', 'manual__2025-03-01T02:09:28.973488+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/dag_generate_table_info.py', '--cfg-path', '/tmp/tmppfzthzt4']
[2025-03-01T02:09:35.259+0000] {standard_task_runner.py:72} INFO - Started process 58605 to run task
[2025-03-01T02:09:35.260+0000] {standard_task_runner.py:105} INFO - Job 87: Subtask generate_orders_data
[2025-03-01T02:09:35.303+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_postgres_1.generate_orders_data manual__2025-03-01T02:09:28.973488+00:00 [running]> on host e6b43e607126
[2025-03-01T02:09:35.384+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='kucikm23@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_postgres_1' AIRFLOW_CTX_TASK_ID='generate_orders_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T02:09:28.973488+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-01T02:09:28.973488+00:00'
[2025-03-01T02:09:35.385+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-01T02:09:35.385+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-01T02:09:35.386+0000] {logging_mixin.py:190} INFO - Current task name:generate_orders_data state:running start_date:2025-03-01 02:09:35.222868+00:00
[2025-03-01T02:09:35.386+0000] {logging_mixin.py:190} INFO - Dag name:dag_postgres_1 and current dag run status:running
[2025-03-01T02:09:35.387+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-01T02:09:35.408+0000] {python.py:240} INFO - Done. Returned value was:                                order_id        customer_email  ... amount  currency
0  062c7d09-7811-4c10-b4e6-22a9c5ca4d8e  user8180@example.com  ...  23.75       BRL

[1 rows x 5 columns]
[2025-03-01T02:09:35.687+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-01T02:09:35.688+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_postgres_1, task_id=generate_orders_data, run_id=manual__2025-03-01T02:09:28.973488+00:00, execution_date=20250301T020928, start_date=20250301T020935, end_date=20250301T020935
[2025-03-01T02:09:35.716+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-01T02:09:35.716+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-01T02:09:35.717+0000] {logging_mixin.py:190} INFO - Dag name:dag_postgres_1 queued_at:2025-03-01 02:09:28.982180+00:00
[2025-03-01T02:09:35.717+0000] {logging_mixin.py:190} INFO - Task hostname:e6b43e607126 operator:PythonOperator
[2025-03-01T02:09:35.755+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-01T02:09:35.775+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-01T02:09:35.779+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
